ipadapter_flux.py

```diff
@@ -113,7 +119,7 @@ class IPAdapterFluxLoader:
     def INPUT_TYPES(s):
         return {"required": {
                 "ipadapter": (folder_paths.get_filename_list("ipadapter-flux"),),
-                "clip_vision": (["google/siglip-so400m-patch14-384"],),
+                "clip_vision": (clip_path_list),
```

Comment:
Minor correction - this needs to be `(clip_path_list,),`. If you don't pass the null second argument, then ComfyUI tries to interpret the path list as the data type.